src/server.c

```diff
@@ -5933,7 +5961,14 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "io_threaded_reads_processed:%lld\r\n"
             "io_threaded_writes_processed:%lld\r\n"
             "reply_buffer_shrinks:%lld\r\n"
-            "reply_buffer_expands:%lld\r\n",
+            "reply_buffer_expands:%lld\r\n"
+            "eventloop_cycles:%lld\r\n"
+            "eventloop_duration_sum:%lld\r\n"
+            "eventloop_duration_max:%lld\r\n"
```

Comment:
I still believe this should be a histogram and not an arbitrary max value that decays.

Reply:
+1. The users usually care more about P99 or P999 latency than average latency, so just exposing sum and count may be not enough, we need more fine grained metrics. Instantaneous metrics somehow mitigates it. Another way is recording all latency at the cost of more memory assumption.

Reply:
OK, a histogram does make more sense, so should we have a histogram for every type of duration we measured?

Reply:
OK, a histogram does make more sense, is it possible to just insert this into `LATENCY HISTOGRAM`?

Reply:
OK, a histogram does make more sense..

Reply:
I agree with Madlyn, that histograms would be a lot more useful. If INFO output size is a concern, should we just add a separate section to the INFO that will not be printed by default and use something like `INFO eventloop` to print this data?